xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java

```diff
@@ -315,6 +309,14 @@ public void endFormat(Format format, Map<String, String> parameters)
         }
     }
 
+    @Override
```

Comment:
I doubt this build.

Reply:
Why?

Reply:
Please build the module, you will get more details in the error...